src/VisualStudio/CSharp/Impl/Debugging/CSharpProximityExpressionsService.RelevantExpressionsCollector.cs

```diff
@@ -136,7 +136,6 @@ public override void VisitSwitchStatement(SwitchStatementSyntax node)
                         {
                             var t = (TupleExpressionSyntax)component;
                             foreach (ArgumentSyntax a in t.Arguments) AddVariableExpressions(a.Expression, expressions);
-                            // TODO REVIEW I think there was a bug there. Are we missing tests?
```

Comment:
Confirmed with Neal there was a bug.
Filed bug https://github.com/dotnet/roslyn/issues/15095 to improve test coverage